docs/design/network-discovery/port-FSM.puml

```diff
@@ -45,7 +45,9 @@ state OPERATIONAL {
     UP : enter / emit port-up into report
     UP : enter[discovery-enabled] / issue addWatch
     UP : poll-discovery / emit discovery
+    UP : poll-discovery[last-discovery-was-unsuccessful] / emit disable slow-discovery
```

Comment:
Where do you track "last-discovery-was-unsuccessful"?

Reply:
Flag change added